### PR TITLE
FIX/AUT-2088 Fix horizontal size choices

### PR DESCRIPTION
**Pull request by meefox**
3/4
Releated to [AUT-2088](https://oat-sa.atlassian.net/browse/AUT-2088)

**Requires:**
* Needed by [tao-core](https://github.com/oat-sa/tao-core/pull/3513)

### CODE REVIEW

**Review by btamas — APPROVED** [review]
**Review Checklist**
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful
- [ ] Pull request's target is not `master`
- [ ] Commits are following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] Commits messages are meaningful
- [x] Commits are atomic
- [ ] Changelog is updated according to changes (if applicable)
- [ ] Documentation is updated according to changes (if applicable)

File: src/util/adaptSize.js [hunk 1]
```diff
@@ -86,6 +86,15 @@ var adaptSize = (function() {
...
         both: function($elements, minWidth, minHeight) {
             _resize($elements, { height: minHeight, width: minWidth });
+        },
+
+        /**
+         * Set height to auto on a set of elements
+         *
+         * @param {jQueryElements} $elements
+         */
+        resetHeight: function($elements) {
```
Comment (jsconan) [inline src/util/adaptSize.js:96]:
```suggestion
        resetHeight($elements) {
```

**Review by jsconan — APPROVED** [review]
The code is ok, even if it could better be ES6.
Any chance to add a unit test for the new method? The others are [tested](https://github.com/oat-sa/tao-core-sdk-fe/blob/5753473b16e07d1f2d1838536c0935fefa3f9117/test/util/adaptSize/test.js).


**Review Checklist**
- [ ] ~~New code is covered by tests (if applicable)~~
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] ~~New code is not subject to concurrency issues (if applicable)~~
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [x] Commits are following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] Commits messages are meaningful
- [x] Commits are atomic

File: src/util/adaptSize.js (hunk 1 above)
Comment (meefox) [reply src/util/adaptSize.js:96]: this will break inconsistency for this file

**Pull request merged**